app/src/components/posts/VideoPlayerVideoMode.css

```diff
@@ -0,0 +1,110 @@
+.video-mode {
+    @apply aspect-video;
+}
+
+.video-mode .vjs-tech {
+    @apply !rounded-lg;
+}
+
+.video-mode .vjs-poster img {
+    @apply !rounded-lg object-cover;
+}
+
+.video-mode .vjs-big-play-button {
+    @apply !border-none bg-zinc-800/60 !text-3xl;
+}
+
+.video-mode .vjs-big-play-button span {
+    @apply !text-5xl;
+}
+
+.video-mode .vjs-control-bar {
+    @apply !relative !h-full !w-full rounded-lg;
+    background-color: rgba(0, 0, 0, 0.3) !important;
+}
+
+.video-mode :is(.vjs-has-started) .vjs-control-bar {
+    @apply block;
+}
+
+.video-mode .vjs-control {
+    @apply !float-left !h-11 !w-11 !text-sm !outline-none;
+}
+
+.video-mode .vjs-picture-in-picture-control {
+    @apply !float-right;
+}
+
+.video-mode .vjs-play-control {
+    @apply !absolute !inset-0 !m-auto !h-16 !w-16 !text-xl;
+}
+
+.video-mode .vjs-progress-control {
+    @apply !absolute !bottom-0 !left-0 !pl-2 !pr-0;
+    width: calc(100% - 40px) !important;
+}
+
+.video-mode .vjs-live-control {
+    @apply !absolute !bottom-0 !left-0 !pl-2 !pr-0;
+}
+
+.video-mode .vjs-live-display {
+    @apply p-3 pl-2;
+}
+
+.video-mode .vjs-slider {
+    @apply !rounded-full;
+}
+
+.video-mode .vjs-slider-bar {
+    @apply !rounded-full;
+}
+
+.audio-mode .vjs-play-progress {
+    @apply !rounded-full;
+}
+
+.audio-mode .vjs-load-progress {
+    @apply !rounded-full;
+}
+
+.video-mode .vjs-fullscreen-control {
+    @apply !absolute !bottom-0 !right-0;
+}
+
+.video-mode .vjs-menu {
+    @apply !left-11 !top-0 !outline-none;
+}
+
+.video-mode .vjs-menu-content {
+    @apply !top-4 !h-fit rounded-md !bg-zinc-50  shadow-lg dark:!bg-zinc-600 dark:text-zinc-100;
+}
+
+.video-mode .vjs-selected {
+    @apply !bg-zinc-300 font-bold focus:!bg-zinc-300 dark:!bg-zinc-500 focus:dark:!bg-zinc-500;
+}
+
+.video-mode .vjs-menu-item-text {
+    @apply text-zinc-900 dark:text-zinc-100;
+}
+
+.video-mode .video-js :not(.vjs-selected) {
+    @apply focus:!bg-transparent;
+}
+
+.video-mode .vjs-menu-button {
+    @apply !outline-none;
+}
+
+.video-mode .vjs-menu-item {
+    @apply !p-3 !text-sm !outline-none;
+    width: 100% !important;
+}
+
+.audio-mode-toggle-video-init {
+    @apply !absolute bottom-2 right-2;
+}
+
+.audio-mode-toggle-video {
+    @apply !absolute bottom-12 right-2;
+}
```

Comment:
I would put all of these in the same file, otherwise it can be hard to find both the audio and video toggle classes in the future. Maybe put these in the `style` tag in the Vue component?